cherab/tools/observers/group/base.py

```diff
@@ -18,33 +18,26 @@
 # under the Licence.
 
 from numpy import ndarray
-import matplotlib.pyplot as plt
 from raysect.core import Node
 from raysect.core.workflow import RenderEngine
-from raysect.optical import Spectrum
-from raysect.optical.observer import SpectralRadiancePipeline0D, SpectralPowerPipeline0D, RadiancePipeline0D
+from raysect.optical.observer import Observer0D
 
 
 class Observer0DGroup(Node):
     """
-    A base class for a group of 0D spectroscopic observers under a single scene-graph node.
+    A base class for handling groups of nonimaging observers as one Node.
 
     A scene-graph object regrouping a series of observers as a scene-graph parent.
     Allows combined observation and display control simultaneously.
     Note that for any property except `names` and `pipelines`, the same value can be shared between
-    all sight lines, or each sight line can be assigned with individual value.
+    all observers, or each observer can be assigned with individual value.
 
     :ivar list names: A list of sight-line names.
-    :ivar list pipelines: A list of all pipelines connected to each sight-line in the group.
-    :ivar list/Point3D origin: The origin points for the sight lines.
-    :ivar list/Vector3D direction: The observation directions for the sight lines.
+    :ivar list pipelines: A list of all pipelines connected to each observer in the group.
     :ivar list/RenderEngine render_engine: Rendering engine used by the sight lines.
                                            Note that if the engine is shared, changing its
                                            parameters for one sight line in a group will affect
                                            all sight lines.
-    :ivar list/bool display_progress: Toggles the display of live render progress.
-    :ivar list/bool accumulate: Toggles whether to accumulate samples with subsequent
-                                observations.
```

Comment:
This could be problematic as the pipeline base class does not implement display_progress or accumulate. This was ok for the SpectroscopicGroup which was inteded for spectroscopic pipelines which have these implemented.

Reply:
Agreed. Let's remove this, it's not a problem to loop over the observer's pipelines and toggle these parameters manually if needed.